client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -49,11 +49,61 @@
 
     private final int maxOutstandingCheckpointRequest;
 
+    private final boolean subscriberForRetention;
+    private final boolean autoPublishAtLastCheckpoint;
+
+    public enum ReaderGroupRetentionConfig {
+        /**
+         * There is no retention done by this {@link ReaderGroup}.
```

Comment:
Please explain in the comment what this value means. We need to explain that this option should be used to create a non-subscriber RG

Reply:
Updated javadoc. Added links to the PDP too.